### PR TITLE
HTML tag fix

### PR DESCRIPTION
**Pull request by NoLogo**
@meshy @adam-incuna Please review?

### CODE REVIEW

**Comment by meshy** [comment]
\o/

File: groups/templates/groups/group_detail_base.html [hunk 1]
```diff
@@ -22,6 +22,7 @@
                 <li><a href="{{ discussion.get_absolute_url }}">{{ discussion.name }}</a></li>
```
Comment (adam-incuna) [inline groups/templates/groups/group_detail_base.html]: This line probably wants to be above the others at its level.

File: groups/templates/groups/group_detail_base.html [hunk 2]
```diff
@@ -22,6 +22,7 @@
...
             </ul>
         {% endfor %}
+        </li>
```
Comment (adam-incuna) [inline groups/templates/groups/group_detail_base.html]: Looking at the whole file, this `</li>` and its matching opening tag want to be inside the `{% for %}` tags:

```
<ul>
    {% for blah in blah %}
        <li>
            <ul> ... </ul>
        </li>
    {% endfor %}
</ul>

**Comment by adam-incuna** [comment]
@NoLogo ah apologies, was commenting while @meshy merged it!  You might want to do a new PR for the changes I suggested, shouldn't take long.

**Pull request merged**